src/lib/tls/asio/asio_stream.h

```diff
@@ -110,6 +115,7 @@ class Stream
       const next_layer_type& next_layer() const { return m_nextLayer; }
       next_layer_type& next_layer() { return m_nextLayer; }
 
+      // TODO: since we are using beast already, use beast::get_lowest_layer for beast::tcp_stream compatibility
```

Comment:
This is needed to make the beast file server (now moved to another repo) work. But we can leave it for another PR.